HTTP2-Core.package/H2PushPromiseFrameReceiver.class/instance/receive..st

```diff
@@ -0,0 +1,7 @@
+receiving
+receive: aFrame
+	"overwrites the H2ReceivingState implementaion as the PushPromise States don't need the lastFlags property
+	and should preserve them for when they return to the normal ReceivingStates.
```

Comment:
"overwrites the H2ReceivingState implementation as the PushPromise States don't need the lastFlags property
and should preserve them for when they return to the normal ReceivingStates.
Also this can't be the last frame in a stream so no endStream handling necessary"
Rechtschreibung; des weiteren: Soll der Kommentar dort bleiben?

Reply:
Der sollte bleiben da es ist für Leute, die das nicht geschrieben haben sonst schwer nachzuvollziehen ist, warum die PushStates ein stark reduziertes receive haben. Falls die Erklärung unklar ist würde ich eher eine Erweiterung oder Modifizierung vorschlagen.